pkg/auth/docker/client.go

```diff
@@ -22,13 +22,6 @@ type Client struct {
 // All changes will only be written to the first config file.
 func NewClient(configPaths ...string) (auth.Client, error) {
 	var configs []*configfile.ConfigFile
-	for _, path := range configPaths {
-		cfg, err := loadConfigFile(path)
-		if err != nil {
-			return nil, errors.Wrap(err, path)
-		}
-		configs = append(configs, cfg)
-	}
 	if len(configs) == 0 {
```

Comment:
I think this will always be 0 .. should we check `len(configPaths)` instead ?

Reply:
Oh, sorry! I've fixed it. Thank you for catching this.